src/common/config/env.validation.ts

```diff
@@ -165,6 +165,10 @@ export class EnvironmentVariables {
   @IsInt()
   @Transform(({ value }) => parseInt(value, 10))
   UPDATE_VALIDATORS_INTERVAL_MS = 10000;
+
+  @IsOptional()
+  @IsString()
+  LIDO_LOCATOR_ADDRESS = '';
```

Comment:
I want to understand why we started using ENV variable instead constant from `lido-nestjs` package

Reply:
This is for testing on a separate network